code-challenges.test.js

```diff
@@ -14,54 +14,341 @@
+// create a new variable called combineNumbers and add the first and second numbers together then .push to the output variable.
+// return output
+
+
+
+// Jest Test:
+
+describe("fibonacciCreator", () => {
+    const fibLength1 = 6
+    const fibLength2 = 10
+    it("return an array that length containing the numbers of the Fibonacci sequence" , () => {
+        expect(fibonacciCreator(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibonacciCreator(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
+// Jest Test 1 Results:
+
+// FAIL  ./code-challenges.test.js
+// fibonacciCreator
+//   ✕ return an array that length containing the numbers of the Fibonacci sequence
+
+// ● fibonacciCreator › return an array that length containing the numbers of the Fibonacci sequence
+
+//   ReferenceError: fibonacciCreator is not defined
+
+//     25 | describe("fibonacciCreator", () => {
+//     26 |     it("return an array that length containing the numbers of the Fibonacci sequence" , () => {
+//   > 27 |         expect(fibonacciCreator(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//        |         ^
+//     28 |         expect(fibonacciCreator(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//     29 |     })
+//     30 | })
+
+//     at Object.expect (code-challenges.test.js:27:9)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
+// Jest Test 2 Results:
+
+// FAIL  ./code-challenges.test.js
+// fibonacciCreator
+//   ✕ return an array that length containing the numbers of the Fibonacci sequence (3 ms)
+
+// ● fibonacciCreator › return an array that length containing the numbers of the Fibonacci sequence
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: [1, 1, 2, 3, 5, 8]
+//   Received: undefined
+
+//     25 | describe("fibonacciCreator", () => {
+//     26 |     it("return an array that length containing the numbers of the Fibonacci sequence" , () => {
+//   > 27 |         expect(fibonacciCreator(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//        |                                              ^
+//     28 |         expect(fibonacciCreator(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//     29 |     })
+//     30 | })
+
+//     at Object.toEqual (code-challenges.test.js:27:46)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
+// Jest Test 3 Results:
+
+// PASS  ./code-challenges.test.js
+// fibonacciCreator
+//   ✓ return an array that length containing the numbers of the Fibonacci sequence (2 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+
+
+// const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+// const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+
+
 // b) Create the function that makes the test pass.
 
+const fibonacciCreator = (array) => {
+    let firstNumber = 0, secondNumber = 1, output = [] 
+    output.push(secondNumber)
```

Comment:
Great pseudocode! Very detailed on how you planned to approach this problem. I would suggest if you have first two values and want them in an array, you could assign them to a new variable. Something like let newArray = [ 0 , 1]